ui/qmake_qmake_immediate.qrc

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE RCC><RCC version="1.0">
```

Comment:
Are those files needed for the scaling?

Reply:
i think i must fix gitignore. That is trash too/